lib/methods/get-orders/response-builder.js

```diff
@@ -0,0 +1,24 @@
+const buildGetOrderssResponse = response => {
+  const getItem = item => ({
+    id: Number(item.id.$value),
+    status: Number(item.status.$value),
+    customerReference: String(item.customerReference.$value)
+  });
+
+  let orders =
+    response && response.return && response.return.orders
+      ? response.return.orders.item
+      : null;
```

Comment:
It might be better to default it to `[]` instead of `null`. If the caller doesn't have any orders in its account, the object currently returned by the `buildGetOrdersResponse` is:

```javascript
{ orders: undefined, page: 1, remainingRecords: false }
```